app/js/main.js

```diff
@@ -54,7 +54,7 @@ var width  = 640,
     // Status bar
     statusBarCurMode   = document.querySelector("#currentMode span"),
     statusBarCurFrame  = document.querySelector("#currentFrame span"),
-    statusBarFrameNum  = document.querySelector("#noOfFrames"),
+    statusBarFrameNum  = document.querySelector("#num-of-frames span"),
```

Comment:
`#num-of-frames span` is an id that is in lower case with hyphens and doesn't match the the other ids used in the status bar. Would you recommend the the other ids in the status bar are changed to match this format?